### ISSUE TITLE
Why is the order of weight assignment different from the order of stack?

### ISSUE THREAD

**Issue by GrayVictoria** [CFM-MSG/Code_URMF#3]
In late_fusion.py Line 90-92:
weight=torch.softmax(torch.stack([img_var, txt_var]), dim=0)
img_w=weight[1]
txt_w=weight[0]

Why is the order of weight assignment different from the order of stack?
Is this more conducive to subsequent feature fusion?
